test/scapy/layers/test_dhcp6.py

```diff
@@ -0,0 +1,1940 @@
+import pytest
+from scapy.all import *
+
+
+# DUID_LLT basic instantiation
+def test_DUIDLLTbasicinstantiation():
```

Comment:
Don't you think it would be cleaner to have the comment as a docstring rather than a comment?